bin/xbps-create/main.c

```diff
@@ -1050,10 +1050,14 @@ main(int argc, char **argv)
 	if (ar == NULL)
 		die("cannot create new archive");
 	/*
-	 * Set compression format, xz if unset.
+	 * Set compression format, zstd by default.
 	 */
-	if (compression == NULL || strcmp(compression, "xz") == 0) {
+	if (compression == NULL || strcmp(compression, "zstd") == 0) {
 		archive_write_add_filter_xz(ar);
```

Comment:
This is still `_xz`.

Reply:
omg you are right, I forgot to change this, thanks